FitNesseRoot/DbFit/AcceptanceTests/JavaTests/HsqlTests/StandaloneFixtures/SetUp/content.txt

```diff
@@ -0,0 +1,9 @@
+|Import Fixture|
```

Comment:
I think this should be paired with following in `TearDown`:
```
!|dbfit.util.ExportFixture|
|dbfit.fixture|
```

This is might be leading to the errors raised during the CI build.